content/es/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -0,0 +1,781 @@
+entrará en terminación de emergencia.
+En este caso, todos los contenedores restantes en el Pod se terminarán
+simultáneamente con un período de gracia corto.
+
+Igualmente, si el Pod tiene un hook `PreStop` que excede el périodo de gracia de
+finalización, puede ocurrir una terminación de emergencia.
+En general, si has usado hooks de `preStop` para controlar el orden de
+terminación sin contenedores sidecar, puedes quitarlos y permitir que el kubelet
+los administre automáticamente.
+{{</note>}}
+
+1. Cuando expira el tiempo de gracia,
+   el kubelet lanza un apagado forzado.
+   El tiempo de ejecución del contenedor envía una señal `SIGKILL`a cualquier
+   proceso que se esté ejecutando en cualquier contenedor en el Pod.
+   El kubelet también limpia un contenedor `pause` escondido si ese tiempo de
+   ejecución del contenedor usa uno.
+1. El kubelet hace una transición del Pod a fase terminal
+   (`Failed` o `Succeeded`, dependiendo del estado de sus contenedores).
+   Este paso está garantizado desde la versión 1.27.
+1. El kubelet lanza la eliminación forzada del objeto Pod del servidor API,
+   estableciendo el período de gracia a 0 (detención inmediata).
+1. El servidor API borra el objeto API del Pod,
+   que ya no es visible desde ningún cliente.
+
+### Terminación Forzada del Pod {#pod-termination-forced}
+
+{{< caution >}}
+Eliminaciones forzadas pueden ser potencialmente disruptivas para algunas cargas
+de trabajo y sus Pods.
+{{< /caution >}}
+
+Por defecto, todas las eliminaciones tienen un tiempo de gracia de 30 segundos.
+El comando `kubelet delete` soporta la opción `--grace-period=<segundos>` que
+permite sobreescribir el valor por defecto y especificar tu propio valor.
+
+Establecer el período de gracia a `0` elimina de forma forzada e inmediata el
+Pod del servidor API.
+Si el Pod aún se está ejecutando en un nodo, esa eliminación forzada hace que
+el kubelet inicie una limpieza inmediata.
+
+{{< note >}}
+Debes especificar una opción adicional `--force` junto con `--grace-period=0`
+para realizar eliminaciones forzadas.
+{{< /note >}}
+
+Cuando se realiza una eliminación forzada,
+el servidor API no espera la confirmación del kubelet de que el Pod ha terminado
+en el nodo en que se está ejecutando.
+Este elimina el Pod en la API inmediatamente para que se pueda crear un Pod con
+el mismo nombre.
+En el nodo, los Pods que están por terminar inmediatamente aún pueden tener un
+pequeño período de gracia antes de ser eliminados de forma forzada.
+
+{{< caution >}}
+La eliminación inmediata no espera la confirmación de que el recurso en ejecución
+ha terminado.
+El recurso puede continuar ejecutándose en el clúster de forma indefinida.
+{{< /caution >}}
+
+Si necesitas eliminar Pods por la fuerza y son parte de un `StatefulSet`,
+mira la documentación
+para [borrar Pods de un StatefulSet](/docs/tasks/run-application/force-delete-stateful-set-pod/).
+### Recolección de elementos no utilizados de los Pods {#pod-garbage-collection}
```

Comment:
The whole section `### Pod shutdown and sidecar containers {##termination-with-sidecars}` is missing:

```suggestion
### Pod shutdown and sidecar containers {##termination-with-sidecars}
...

### Recolección de elementos no utilizados de los Pods {#pod-garbage-collection}
```